### PR TITLE
Features/more results page

### PR DESCRIPTION
**Pull request by gsiri-code**

### CODE REVIEW

File: app/static/new-results-page/style-new-results-page.css [hunk 1]
```diff
@@ -0,0 +1,448 @@
...
+}
+
+.header__item__name {
+    font-size: 48px;
+    line-height: 60px;
+    padding: 10px 0;
+
+}
+
+#arrow {
```
Comment (1mpossible-code) [inline app/static/new-results-page/style-new-results-page.css]: Using ids in css is a bad practice.

File: app/static/new-results-page/style-new-results-page.css [hunk 2]
```diff
@@ -0,0 +1,448 @@
...
+    height: 70px;
+
+}
+
+.results-table__header__text {
+    font-size: 16px;
+
+}
+
+#title-static-text {
```
Comment (1mpossible-code) [inline app/static/new-results-page/style-new-results-page.css]: Using ids in css is a bad practice.

File: app/static/new-results-page/style-new-results-page.css [hunk 3]
```diff
@@ -0,0 +1,448 @@
...
+    position: relative;
+    right: 20px;
+    transition: .3s;
+}
+
+#arrow:hover {
+    scale: 1.3;
+}
+
+#dumpling-container {
```
Comment (1mpossible-code) [inline app/static/new-results-page/style-new-results-page.css]: Using ids in css is a bad practice.

File: app/static/new-results-page/style-new-results-page.css [hunk 4]
```diff
@@ -0,0 +1,448 @@
...
+
+}
+
+#arrow {
+    position: relative;
+    right: 20px;
+    transition: .3s;
+}
+
+#arrow:hover {
```
Comment (1mpossible-code) [inline app/static/new-results-page/style-new-results-page.css]: Using ids in css is a bad practice.

File: app/static/new-results-page/style-new-results-page.css [hunk 5]
```diff
@@ -0,0 +1,448 @@
...
+        scale: 1.2;
+    }
+
+    .count-container {
+        position: relative;
+        left: 20px;
+        bottom: 10px;
+    }
+
+    #dumpling-container {
```
Comment (1mpossible-code) [inline app/static/new-results-page/style-new-results-page.css]: Using ids in css is a bad practice.

File: app/static/new-results-page/style-new-results-page.css [hunk 6]
```diff
@@ -0,0 +1,448 @@
...
+        left: 230px;
+        bottom: 20px;
+    }
+
+    #dumpling-container > img {
```
Comment (1mpossible-code) [inline app/static/new-results-page/style-new-results-page.css]: Using ids in css is a bad practice.

Comment (1mpossible-code) [reply app/static/new-results-page/style-new-results-page.css]: And try to create a specific class for image

File: app/static/new-results-page/style-new-results-page.css [hunk 7]
```diff
@@ -0,0 +1,448 @@
...
+    }
+
+}
+
+@media (max-width: 768px) {
+    header{
+        flex-direction: column;
+
+    }
+    #dumpling-container{
```
Comment (1mpossible-code) [inline app/static/new-results-page/style-new-results-page.css]: Using ids in css is a bad practice.

File: app/templates/new-results-page.html [hunk 8]
```diff
@@ -0,0 +1,803 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="{{ url_for('static', filename='new-results-page/style-new-results-page.css') }}">
+    <script src="https://kit.fontawesome.com/8e9929512e.js" crossorigin="anonymous"></script>
+
+    <title>Title</title>
```
Comment (1mpossible-code) [inline app/templates/new-results-page.html]: Make a custom title please

File: app/templates/new-results-page.html [hunk 9]
```diff
@@ -0,0 +1,803 @@
...
+</head>
+
+<body>
+
+<main class="results-area">
```
Comment (1mpossible-code) [inline app/templates/new-results-page.html]: Create a better name for class results-area

**Review by 1mpossible-code — CHANGES_REQUESTED** [review]

File: app/static/new-results-page/style-new-results-page.css [hunk 10]
```diff
@@ -97,24 +104,16 @@ header {
 
 }
 
-#arrow {
+[class="fa-solid fa-arrow-left"] {
```
Comment (1mpossible-code) [inline app/static/new-results-page/style-new-results-page.css]: Fix notation

File: app/static/new-results-page/style-new-results-page.css [hunk 11]
```diff
@@ -97,24 +104,16 @@ header {
...
     position: relative;
     right: 20px;
     transition: .3s;
 }
 
-#arrow:hover {
+[class="fa-solid fa-arrow-left"]:hover {
```
Comment (1mpossible-code) [inline app/static/new-results-page/style-new-results-page.css]: Bad notation

File: app/templates/new-results-page.html [hunk 12]
```diff
@@ -8,18 +8,18 @@
     <link rel="stylesheet" href="{{ url_for('static', filename='new-results-page/style-new-results-page.css') }}">
     <script src="https://kit.fontawesome.com/8e9929512e.js" crossorigin="anonymous"></script>
 
-    <title>Title</title>
+    <title>Nourish:More Results</title>
```
Comment (1mpossible-code) [inline app/templates/new-results-page.html]: Add space

**Review by 1mpossible-code — CHANGES_REQUESTED** [review]

File: app/templates/new-results-page.html (hunk 12 above)
Comment (gsiri-code) [reply app/templates/new-results-page.html]: > Add space

where 💀💀💀

Comment (1mpossible-code) [reply app/templates/new-results-page.html]: between nourish and more results

Comment (gsiri-code) [reply app/templates/new-results-page.html]: > between nourish and more results

ahhh i got it

**Review by 1mpossible-code — APPROVED** [review]

**Pull request merged**